style.css

```diff
@@ -0,0 +1,25 @@
+@media screen and (min-width: 650px) {
+    
+
+}
+#navbar {
+    margin-left: auto;
+    float: left;
+    display: block;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px;
+    text-decoration: none;
+    position: fixed;
+}
+        #navbar {
+            margin-left: auto;
+            float: left;
+            display: block;
+            color: #f2f2f2;
+            text-align: center;
+            padding: 14px;
+            text-decoration: none;
+        }
```

Comment:
Kindly replace "Duplicate your #navbar style" with "Your #navbar rule is duplicated" because it sounds like you are giving orders to duplicate the #navbar rule one more time.